src/test/java/core/basesyntax/service/impl/FileWriterImplTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.IOException;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterImplTest {
+    private static FileWriter fileWriter;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileWriter = new FileWriterImpl();
+    }
+
+    @Test
+    public void fileWriter_usualState_ok() {
+        String expectedOutputPath = "src/test/resources/expectedOutput.csv";
+        String actualOutputPath = "src/test/resources/actualOutput.csv";
+        String data = new StringBuilder()
+                .append("fruit").append(",")
+                .append("quantity").append(System.lineSeparator())
+                .append("apple").append(",")
+                .append("15").append(System.lineSeparator())
+                .toString();
+        try (BufferedWriter bufferedWriter = new BufferedWriter(
+                new java.io.FileWriter(expectedOutputPath))) {
+            bufferedWriter.write(data);
+        } catch (IOException e) {
+            throw new RuntimeException("Cant write data to file" + expectedOutputPath, e);
+        }
+        fileWriter.writeToFile(data, actualOutputPath);
+        StringBuilder stringBuilder = new StringBuilder();
+        String expected;
+        String actual;
+        try (BufferedReader bufferedReader = new BufferedReader(
+                new FileReader(expectedOutputPath))) {
```

Comment:
and read only once - from actual file
you can imagine what content will be in the file because you were writing it